source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,239 @@
+==========================
+1. Halloween Treasure Hunt
+==========================
+
+.. contents:: Table of Contents
+    :local:
+
+Introduction
+============
+
+In this first example project, we'll start with the basics. The following topics will be covered:
+
+* Denizen objects and general syntax.
+* Browsing through the documentation.
+* Reloading scripts and debug modes.
+* World script containers, events and context tags.
+* Common script commands and arguments.
+* Constructing tags with simple bases and modifiers.
+* Basic usage of foreach loops.
+
+If you are not familiar with any of the concepts listed, we suggest revisiting the following tutorial documents:
+
+* :doc:`Section 1.3 - dScript Format </docs/getting-started/dscript-format>`
+* :doc:`Section 2.1 - An Introduction to Tags </docs/basics-of-scripting/an-introduction-to-tags>`
+* :doc:`Section 2.2 - The Basics of Denizen Commands </docs/basics-of-scripting/basics-of-denizen-commands>`
+* :doc:`Section 2.7 - Your First World Script </docs/basics-of-scripting/your-first-world-script>`
+* **[TODO: Add FOREACH section reference]**
+
+This will help you make a better use of this example project, and follow its contents successfully.
+
+.. note::
+    It is also recommended to have the `meta documentation`__ open in another tab while scripting this project, as 
+    we'll need to check the syntax of events, commands and tags very often.
+
+.. __: https://one.denizenscript.com/denizen/logs
+    
+1. First Steps
```

Comment:
I'm a little unsure about this section name. Can it be a little more specific, or less general?